crates/noirc_evaluator/src/ssa/ssa_form.rs

```diff
@@ -92,3 +95,63 @@ pub fn get_current_value_in_block(
 pub fn get_current_value(ctx: &mut SsaContext, var_id: NodeId) -> NodeId {
     get_current_value_in_block(ctx, var_id, ctx.current_block)
 }
+
+fn get_array_size(array_size: &ArraySize) -> u32 {
+    match array_size {
+        ArraySize::Fixed(l) => *l as u32,
+        ArraySize::Variable => todo!(),
+    }
+}
+
+pub fn create_function_parameter(
+    igen: &mut IRGenerator,
+    ident_id: &noirc_frontend::node_interner::IdentId,
+) -> NodeId {
+    let ident_name = igen.context.context().def_interner.ident_name(ident_id);
+    let ident_def = igen.context.context().def_interner.ident_def(ident_id);
```

Comment:
I think there may be a function `igen.def_interner()` or `igen.context.def_interner()` that can be used to simplify this a bit

Reply:
Done.
There was even ident_name() and ident_def() function that I just needed to make public.